apps/website/src/pages/docs/dialog.mdx

```diff
@@ -13,58 +17,60 @@ import LiveExample from '~/components/LiveExample.astro';
 
 <p>{frontmatter.description}</p>
 
-<LiveExample src={frontmatter.exampleCodeFile1} />
+<LiveExample src={frontmatter.exampleCodeMain} />
 
 A dialog informs users about a task and can contain critical information, require decisions, or involve multiple tasks. Dialogs appear in front of app content to provide critical information or ask for a decision. Dialogs disable all app functionality when they appear, and remain on screen until confirmed, dismissed, or a required action has been taken.
 
 Dialogs are purposefully interruptive, so they should be used sparingly.
 
-## Anatomy
+## Dialog VS Modal
 
-- Title: must be descriptive of the task at hand. The user must understand the context because they went from the main window to a dialog.
-- Close button: only allowed in dismissible dialogs.
-- Content
-- Actions: see Button standard for best practices.
-- Box & shadow
-- Overlay: Dialog appears above all other UI elements and casts a black overlay as well as a drop shadow on all other content, creating a strong visual focus on the dialog.
+The terms dialog and modal are often used interchangeably. Within iTwinUI we differentiate the two terms like so:
 
-The dialog should be located in the upper half of the screen, because on smaller screens it may get lost. A dialog’s maximum width is 50% of the screen width & the minimum width is 400px. If your content can not fit inside the dialog without scrollbars, you should consider alternative ways to display the information.
+- Dialog: a dialog is the box of content that appears above the rest of the page content. The user can still interact with the page content.
+- Modal dialog: a modal dialog is the combination of a dialog with a backdrop that appears above the rest of the page content. This backdrop disables interaction with the page content until the dialog is dismissed.
```

Comment:
These changes looks good.
Maybe you could mention that main example is Modal dialog? 
Or change main example to be Dialog with backdrop and add additional example of Modal dialog. 
(I can help you with that if you want)

Reply:
I'd appreciate that, thank you.  Short meeting scheduled for Monday.